CHANGELOG.md

```diff
@@ -2,7 +2,19 @@
 
 librdkafka v1.9.3 is a maintenance release:
 
- *  Self-contained static libraries can now be built on Linux arm64 too
+ * Self-contained static libraries can now be built on Linux arm64 too
+ * Fixes to the transactional and idempotent producer (#3971).
+
+
+## Fixes
+
+### Transactional producer fixes
+
+ * When requesting an abort with drain and bump after message timeouts, the bump was executed first, causing a series of retries on InitProducerIdRequest until a transaction timeout happen and the producer was fenced. This commit changes the order of the operations, starting the drain bump after the complete transaction ack (#3971).
```

Comment:
maybe: "In certain circumstances, an `InitProducerId` request would be issued to bump the producer epoch before the open transaction was aborted, causing the producer to enter a failed state. The producer will now wait until the transaction is aborted before requesting an epoch bump."?